tests/discriminant.rs

```diff
@@ -533,7 +533,7 @@ fn repr_c_expr() {
 	#[derive_where(PartialEq, PartialOrd)]
 	#[repr(C)]
 	enum Test {
-		A = isize::MAX - 2,
+		A = (u32::MAX - 2) as isize,
```

Comment:
Why this change?

Reply:
I think we should stick with `isize` until we get there, we also use `isize` internally, so we should test for that.

(on a side note: I'm doubtful this change will actually happen because it would break too much)

Reply:
Yeah, as this is only tests, we could just disable the lint

Reply:
Yeah, as this is only tests, we could just disable the lint